doc/build/orm/loading_columns.rst

```diff
@@ -83,23 +85,27 @@ column will load immediately even if it was mapped as a "deferred" column.
 using :func:`_orm.undefer_group`, sending in the group name::
 
     from sqlalchemy.orm import undefer_group
+    from sqlalchemy import select
+
+    stmt = select(Book)
+    stmt = stmt.options(undefer_group('photos'))
```

Comment:
add `session.scalars(stmt).all()`

i think it's important the examples show how to get to the end-result of a list of objects

then again I see a bunch of my examples below aren't doing it, maybe this is why people complain so much about the docs :)

Reply:
> i think it's important the examples show how to get to the end-result of a list of objects

I feel like that would be the best approach, and maybe the cases you mentioned would benefit from adhering to this. But as I said elsewhere, this is probably best not done in the scope of this PR, I was a bit overzealous there.

(I can't believe that this is the reason people complain about the docs, since frankly, they're one of the best docs I've ever read / worked with)